learning/src/components/App.js

```diff
@@ -1,13 +1,37 @@
 import React, { Component } from "react";
 import { render } from "react-dom";
+import WordDefinition from "./WordDefinition";
 
 class App extends Component {
     render() {
-        return "Hello World";
+        //change the following constants with actual data
+        const cn = "字";
+        const py = "zi";
+        const src = "here";
+        const arr = [
+            {
+                "part_of_speech": "n.",
+                "definition"    : "a noun \n second line"
+            },
+            {
+                "part_of_speech": "adj.",
+                "definition"    : "an adj"
+            }];
+
+        return (
+            <div>
+                <WordDefinition
+                    word={cn}
+                    pinin={py}
+                    audiosrc={src}
+                    definitions={arr}
+                />
```

Comment:
I would say later it would be better to take in a single argument called word that is like this:
`{
    "url": "https://dev.solvedchinese.org/content/word/1",
    "definitions": [
        {
            "part_of_speech": "adv",
            "definition": "hello"
        }
    ],
    "sentences": [
        {
            "pinyin": "ni hao, wo shi",
            "chinese": "你好, 我是",
            "translation": "hello, i am"
        },
        {
            "pinyin": "ni hao ma",
            "chinese": "你好吗",
            "translation": "hello ma"
        }
    ],
    "note": "",
    "is_done": false,
    "chinese": "你好",
    "identifier": "",
    "pinyin": "nǐ hǎo",
    "memory_aid": "this is memory aid",
    "characters": [
        "https://dev.solvedchinese.org/content/character/2",
        "https://dev.solvedchinese.org/content/character/1"
    ]
}`